pytest_girder/pytest_girder/fixtures.py

```diff
@@ -0,0 +1,126 @@
+import cherrypy
+import hashlib
+import mock
+import pytest
+from pytest_mongodb.plugin import mongo_engine
+
+from .utils import request
+
+
+@pytest.fixture(autouse=True)
+def bcrypt():
+    """
+    Mock out bcrypt password hashing to avoid unecessary testing bottlenecks.
+    """
+    with mock.patch('bcrypt.hashpw') as hashpw:
+        hashpw.side_effect = lambda x, y: x
+        yield hashpw
+
+
+@pytest.fixture
+def db(request, mongodb):
```

Comment:
@jbeezley 
I think we should remove `pytest-mongodb` and "roll it ourselves" for the following reasons:
- We won't be using their fixtures since we have our own fixture system (which is higher level, folders, items, etc)
- It always drops databases before running, breaking the flag we added: `--drop-db={pre,post,never}`.
- It provides its own option for specifying the mongo URI, conflicting with our own

The logic I see it bringing to the table for us can be boiled down to this [8 line function](https://github.com/mdomke/pytest-mongodb/blob/develop/pytest_mongodb/plugin.py#L69).

Reply:
I rewrote the mongo mocking part to use mongomock directly.  You're right, it's much simpler now.